pkg/cloud/provider.go

```diff
@@ -104,6 +104,15 @@ type Network struct {
 	InternetNetworkEgressCost float64
 }
 
+type OrphanedResource struct {
+	Kind        string            `json:"resourceKind"`
+	Region      string            `json:"region"`
+	Description map[string]string `json:"description"`
+	Size        *int32            `json:"diskSizeInGB, omitempty"`
+	Address     string            `json:"ipAddress, omitempty"`
```

Comment:
No space after the comma with these `omitempty`s. That's likely why the omission is not happening.